src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+
+import styled from 'styled-components';
+
+import HomeVideos from '../../components/Home/HomeVideos.component';
+
+const { REACT_APP_API_KEY } = process.env;
+
+function HomePage({ searchTerm }) {
+  const [videos, setVideos] = useState(null);
+
+  const API_URL = `https://www.googleapis.com/youtube/v3/search?key=${REACT_APP_API_KEY}&q=${searchTerm}&part=snippet,id&order=date&maxResults=20`;
+
+  async function searchVideos() {
+    try {
+      const response = await fetch(API_URL);
+      const videosRes = await response.json();
+
+      setVideos(videosRes);
+    } catch (error) {
+      console.error('Something went wrong: ', error);
+    }
+  }
+
+  useEffect(() => {
+    console.log(searchTerm);
+    searchVideos();
+  }, [searchTerm]);
```

Comment:
This works well. A good practice when dealing with API calls that can change so abruptly like this one is to leverage the `debounce` function. It is very useful to avoid having too many calls (which may lead to bigger expenses on some APIs) and also if the connection is unreliable we avoid having weird cache and stale state issues